packages/cli/src/lib/defaults/deploy-modules/ipfs/index.ts

```diff
@@ -18,26 +77,34 @@ class IPFSDeployer implements DeployModule {
     }
 
     const path = uri.path;
+    const data: DirectoryEntry = await readDirContents(path, "");
 
-    const ipfsUrl = config?.gatewayUri ?? "http://localhost:5001";
+    const ipfsProvider = config?.gatewayUri ?? "http://localhost:5001";
 
-    const client = new IPFSClient({ url: ipfsUrl });
-    const globOptions = {
-      recursive: true,
+    const args: ArgsAddDir = {
+      data,
+      ipfsProvider,
+      addOptions: {
+        pin: true,
+        wrapWithDirectory: false,
+      },
+      timeout: 10000,
     };
 
-    const addOptions = {
-      wrapWithDirectory: false,
-    };
+    const client = new PolywrapClient();
 
-    let rootCID = "";
+    const result = await client.invoke<AddResult[]>({
```

Comment:
@krisbitney I'm good with merging this now, but can you please make a TODO for an A/B test locally? I want to make sure that deploying a wrap using this new ipfs deployer yields the same URI as the old deployer.

Reply:
Sure thing. It produces the same IPFS URI in the CLI test cases. I'll make sure it also works with a real wrap.

Reply:
I confirmed that it produces the same IPFS hash for a real wrap